docs/reference/glossary.md

```diff
@@ -0,0 +1,10 @@
+Since so many terms are thrown around in the documentation and comments within the codebase below is a short glossary to ensure everyone is talking about the same thing.
```

Comment:
```suggestion
# Glossary

Since so many terms are thrown around in the documentation and comments within the codebase below is a short glossary to ensure everyone is talking about the same thing.
```